src/core/public/notifications/toasts/toasts_api.tsx

```diff
@@ -145,6 +145,22 @@ export class ToastsApi implements IToasts {
     }
   }
 
+  /**
+   * Adds a new toast pre-configured with the info color and info icon.
+   *
+   * @param toastOrTitle - a {@link ToastInput}
+   * @param toastLifeTimeMs - a {@link number}
+   * @returns a {@link Toast}
+   */
+  public addInfo(toastOrTitle: ToastInput, toastLifeTimeMs?: number) {
```

Comment:
`toastLifeTimeMs` isn't an option for any other method here. For consistency, I think we should either add it to all methods or simply require that the plugin dismiss it with a `setTimeout`.

If you do decide it to all, can we add it as an options object to make future extension easier:

```ts
public addInfo(toastOrTitle: ToastInput, options: { toastLifeTimeMs?: number } = {})
```

Reply:
Done!